### DISCUSSION TITLE
Change subtile styling by default

### DISCUSSION THREAD

**Discussion by RobertArissen** [TheWidlarzGroup/react-native-video#4942] [Ideas]
### What problem are you trying to solve?

I would like to show subtitle text without a background. There doesn't appear to be a way to pass a standard subtitle style to the player, and I couldn't find one.

### Proposed solution

Ability to pass a standard subtitle style to the player so the default subtitle styling can be changed (for example, displaying subtitle text without a background).

### Additional context

The author notes they are not sure if this is possible at all.

**Comment by freeboub** [comment]
This feature is not supported, but here is a clue for android:

You should be able to achieve this by patch the code by adding something like follwoing line :

subView.setStyle( new CaptionStyleCompat( Color.WHITE, Color.TRANSPARENT, Color.TRANSPARENT, CaptionStyleCompat.EDGE_TYPE_NONE, Color.WHITE, null ) );

in ExoPlayerView.java, near the view instanciation line 78.

**Comment by RobertArissen** [comment]
Do you also find a solution to fix this for Tvos?

**Comment by TWG-AI** [comment]
Thanks for the idea — noted for the team to consider.

> I'm an AI assistant, so double-check any specifics I may have mentioned.
